.github/workflows/test.yml

```diff
@@ -53,13 +50,11 @@ jobs:
         run: gradle test ${RUNNER_DEBUG:+"--debug"}
 
       - name: SonarQube analysis
-        run: gradle sonarqube -Dsonar.projectKey=$SONAR_PROJECT_KEY -Dsonar.host.url=$SONAR_HOST_URL
+        run: gradle sonarqube -Dsonar.host.url=$SONAR_HOST_URL
```

Comment:
Does `gradle sonar` work here? That should take care of this deprecation warning:

https://github.com/CMSgov/ab2d-events/actions/runs/5201241566/jobs/9381098978#step:7:30